karma.conf.ts

```diff
@@ -72,7 +72,7 @@ function makeBuildNumber() {
 }
 
 function setupLocal(config: Config) {
-  const ciSpecificFiles = ['tests/utils/karma_global_setup_retries.ts']
+  const ciSpecificFiles = ['config/karma/karma_global_setup_retries.ts']
```

Comment:
Creating a root-level directory for a single file is too much. Let's use `resources/karma` instead?

Reply:
I was thinking we could use this directory for similar configs